stylesheet.css

```diff
@@ -297,15 +303,6 @@ hr {
   padding: 12px;
 }
 
-.seep {
-  display: flex;
-  justify-content: left;
-  height: 10px;
-  width: 327px;
-  padding-bottom: 10px;
-  padding-left: 0.000001px;
-}
-
 .seeproject:active {
```

Comment:
- [ Optional ] For Mobile screen size I recommend you to adjust alignment of the "See Project" button just like the figma design. 

Your Project content Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/200140637-18b5ff31-5b9c-48be-8f89-08a8f7e00ed7.PNG)


The Figma Project content Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/200140683-c39b82f7-0bec-478b-92ad-5ea31d595de7.PNG)